packages/default-storage/windows/ReactNativeAsyncStorage/ReactPackageProvider.cpp

```diff
@@ -0,0 +1,20 @@
+#include "pch.h"
+
+#include "ReactPackageProvider.h"
+#if __has_include("ReactPackageProvider.g.cpp")
+#include "ReactPackageProvider.g.cpp"
+#endif
+
+#include "ReactNativeAsyncStorage.h"
+
+using namespace winrt::Microsoft::ReactNative;
+
+namespace winrt::ReactNativeAsyncStorageAsyncStorage::implementation
+{
+
+void ReactPackageProvider::CreatePackage(IReactPackageBuilder const &packageBuilder) noexcept
+{
+  AddAttributedModules(packageBuilder, true);
```

Comment:
Whats the supported RNW version range of async-storage?  -- Should we put the extra "true"  args behind a version check?

Reply:
Actually, I think this file is unused - the included file is https://github.com/react-native-async-storage/async-storage/pull/1147/files/b9f2e55486dfcbeb15f9dc6d0ab37e999913a553#diff-91c8b85a4a6cfde2508040988d613ec059855d3e4cca40423d0edc7478af9301

I'm using `USE_FABRIC` to decide whether to use the `true` arg or not.